docs/api/breaking-changes.md

```diff
@@ -64,6 +64,17 @@ webFrame.setIsolatedWorldInfo(
   })
 ```
 
+### `autoUpdater.setFeedURL`
```

Comment:
Can these be moved to _at least_ planned to break in Electron 8 🤔 preferably 9 now we're on the 12 week cycle

Reply:
yeah, i think we actually should just centralize that info in the `property-updates.md` file and not put it in `breaking-changes.md`

Reply:
They should be in a new 8.0 section instead of the 7.0 section